### PR TITLE
Added a new blog on providing feedback.

### PR DESCRIPTION
**Pull request by Bhargav-Rao**

### CODE REVIEW

File: _posts/2019-01-22-providing-feedback-to-bots.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Why and How do we provide feedback to the bots?"
+date: 2019-01-22 7:30:00 +0000
+comments: false
+---
+
+### Why do we need to feedback to bots? 
+
+We send feedback to the bots for us to know which all reports have already been handled. We also collect some statistics and  improve the filters based on this feedback. Few of the bots are built on ML and hence they need feedback to improve.
```
Comment (ArtOfCode-) [inline _posts/2019-01-22-providing-feedback-to-bots.md]: > We send feedback to the bots to let us know which all reports have already been handled. We also collect some statistics and  improve the filters based on this feedback. Some of the bots are built on ML and hence they need feedback to improve.

File: _posts/2019-01-22-providing-feedback-to-bots.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+title: "Why and How do we provide feedback to the bots?"
+date: 2019-01-22 7:30:00 +0000
+comments: false
+---
+
+### Why do we need to feedback to bots? 
+
+We send feedback to the bots for us to know which all reports have already been handled. We also collect some statistics and  improve the filters based on this feedback. Few of the bots are built on ML and hence they need feedback to improve. 
+
+The continual improvement of the bots is one way to see how your feedback has affected the bot. Soon we are planning to utilize SOPlotics to plot nice graphs and figures for the same.
```
Comment (ArtOfCode-) [inline _posts/2019-01-22-providing-feedback-to-bots.md]: > The continuous improvement of the bots is one way to see how your feedback has affected the bot. We are also planning to start using SOPlotics to generate graphs and visualisations of this.

File: _posts/2019-01-22-providing-feedback-to-bots.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+### What all bots  take feedback? 
+
+ - Belisarius posts possible vandalized posts and takes `tp` or `fp` as feedback.
+ - Guttenberg posts possible plagiarized answers and takes `k` or `f` (and `tp` or `fp`) as feedback. See [How to react to Guttenberg's reports?](https://github.com/SOBotics/Guttenberg/blob/master/feedback.md)
+ - HeatDetector posts possible inflammatory comments and takes `tp`, `fp`, `fn`, `nc` and `sk` as feedback. See [What all types of feedback does Heat Detector take?](https://github.com/SOBotics/HeatDetector/wiki/Feedback)
+ - Natty posts possible non-answers and takes `tp`, `fp` and `ne` as feedback. See [How do I interact with Natty?](https://natty.sobotics.org/interacting). 
+ - SOCVFinder posts possible duplicate questions and takes `k` or `f` as feedback. 
+
+Additionally, our guest, SmokeDetector, posts possible spam or rude or abusive posts, and it takes `tp` (`k`, `v`), `fp` (`f`) or `naa` (`n`) as feedback via replying to its report messages.
```
Comment (ArtOfCode-) [inline _posts/2019-01-22-providing-feedback-to-bots.md]: Add: "See [Feedback Guidance](https://charcoal-se.org/smokey/Feedback-Guidance) on the Charcoal website for details."

**Review by ArtOfCode- — COMMENTED** [review]
Some grammar suggestions, mostly.

**Comment by Bhargav-Rao** [comment]
@ArtOfCode- made the changes requested, thank you.

**Pull request merged**